src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarAuctionPartnerInfo.tsx

```diff
@@ -21,14 +21,6 @@ export class ArtworkSidebarAuctionPartnerInfo extends React.Component<
     subject: Schema.Subject.AuctionBuyerPremium,
     type: Schema.Type.Link,
   }))
-  onClickBuyerPremium(mediator) {
```

Comment:
Can we remove the query for the  `is_with_buyers_premium` field in the [graphql query for this component](https://github.com/artsy/reaction/pull/2966/files?utf8=%E2%9C%93&diff=unified&w=1#diff-c9ef16c3b323f531057622da216d4a53R52-R68)? I believe it's no longer needed now.